ggdeploymentd/src/deployment_handler.c

```diff
@@ -1155,6 +1156,16 @@ static GglError resolve_dependencies(
             component_version = val->buf;
         }
 
+        if (ggl_buffer_eq(pair->key, GGL_STR("aws.greengrass.NucleusLite"))) {
+            if (!ggl_buffer_eq(component_version, GGL_STR(GGL_VERSION))) {
+                GGL_LOGE("The version of the aws.greengrass.NucleusLite "
+                         "component does not "
+                         "match the already installed version. "
```

Comment:
I think this should specify the version of the running GG Lite, since saying it doesn't match the installed version would make it sound like the software is deployed similar to how classic is which isn't true